### ISSUE TITLE
Not receiving group chats from Facebook

### ISSUE THREAD

**Issue by mihav** [hanzz/spectrum2#173]
Hello,
I'm using latest ejabberd+spectrum2+purple-facebook combination on server side with Gajim and Conversations on client side. Everything works pretty well, except Facebook groupchat messages are not delievered.

Log file says:
==> backends/backend-1.log <==
26379: 2016-11-22 19:57:37,506 INFO  backend: Received message body='message' xhtml='' name='569408136428298' 1044469493

==> spectrum2.log <==
2016-11-22 19:57:37,508 WARN  NetworkPluginServer: handleConvMessagePayload: No conversation with name 569408136428298

**Comment by mw80** [comment]
I have the same issue. Any update here?

**Comment by mw80** [comment]
Maybe group-chat-open parameter must be set to 1. But currently not possible to test as backend is crashing.

[purple]
group-chat-open=1

**Comment by vitalyster** [comment]
can you please test https://github.com/hanzz/spectrum2/commit/898b05c61506e78c3980f5c6f80670f7e3c0098b ?

**Comment by mihav** [comment]
Nope, still not working with group-chat-open=1 with latest master branch:

10062: 2017-01-27 10:17:45,410 ERROR backend: g_log purple_conversation_get_chat_data: assertion 'conv != NULL' failed
10062: 2017-01-27 10:17:45,411 INFO  libpurple: http:Performing new request 0xab85f0 to graph.facebook.com.
10062: 2017-01-27 10:17:45,411 INFO  libpurple: http:Connecting to graph.facebook.com...
10062: 2017-01-27 10:17:45,411 INFO  backend: Received message body='test group message' xhtml='' name='1420768124622546' $USERID

I also noticed messages which only contains smileys (only thumbsup for example) are not coming through. Spectrum logs empty message:

10062: 2017-01-27 10:20:22,353 ERROR backend: GLib g_string_free: assertion 'string != NULL' failed
10062: 2017-01-27 10:20:22,353 INFO  backend: Received message body='' xhtml='' name='$USERID'

**Comment by thundergreen** [comment]
I see also participants but messages not delivered will try above and give feedback

**Comment by vitalyster** [comment]
So, at least something received from facebook side, but not delivered to XMPP. That is a backend log, can you also please check spectrum log?

**Comment by mihav** [comment]
spectrum2 log for incoming group chat (same as in first report):

2017-01-27 10:17:45,412 WARN  NetworkPluginServer: handleConvMessagePayload: No conversation with name 1420768124622546
2017-01-27 10:17:46,166 WARN  NetworkPluginServer: handleConvMessagePayload: No conversation with name 1420768124622546

**Comment by thundergreen** [comment]
Tried adding mentioned  prpl settings but spectrum won't start

**Comment by vitalyster** [comment]
@thundergreen you forgot to update spectrum

**Comment by thundergreen** [comment]
I will have to build it from Scratch...  Would do that if you would release
my Pckages  :)

Am 27.01.2017 14:56 schrieb "vitalyster" <notifications@github.com>:

**Comment by mw80** [comment]
I got the same error also with the new version when someone is inviting my fb account to a group chat:
group-chat-open=1

2017-01-27 21:44:29,914 WARN  NetworkPluginServer: handleConvMessagePayload: No conversation with name 1283337048421837
2017-01-27 21:44:30,149 WARN  NetworkPluginServer: handleConvMessagePayload: No conversation with name 1283337048421837

**Comment by truenicoco** [comment]
Any update on this? I can manually join an existing groupchat, but isn't there a way to auto "popup" new group chats? Also, is there a way to create a facebook groupchat using a xmpp client?

**Comment by vitalyster** [comment]
@truenicoco @Knezzen if you are able to join/receive messages, then it is a different issue and need a lot of work on server/spectrum AND client side - https://github.com/SpectrumIM/spectrum2/issues/68
